spec/graphql/mutations/wallets/create_spec.rb

```diff
@@ -31,7 +31,7 @@
       query: mutation,
       variables: {
         input: {
-          customerId: customer.id,
+          customerId: customer.customer_id,
```

Comment:
Not sure that it will be easy for front-end to know which customer_id to use :thinking: 

My point is we should limit the usage of this field to public API. Open to discuss that point though :)

Reply:
We already synced with Alex and it's used on FE for this feature. For me it's also a bit easier to use `customer_id` since I can reuse the same service from api controller in that case :)

Reply:
we already use `customer_id` on mutations linked to the customer (create subscriptions etc), maybe it's better to keep it this way for consistency

Reply:
Sorry to bring back this one, but no on the front, we are using id:
- https://github.com/getlago/lago-api/blob/main/app/services/customers/update_service.rb#L6
- https://github.com/getlago/lago-api/blob/main/app/services/subscriptions/create_service.rb#L32

That's why I was bringing the consistency point